src/leap/mail/imap/service/imap.py

```diff
@@ -164,6 +218,7 @@ def run_service(store, **kwargs):
         if os.environ.get("LEAP_DOCKERIZED"):
             interface = ''
 
+        # TODO use Endpoints !!!
```

Comment:
https://twistedmatrix.com/documents/current/core/howto/endpoints.html
nowadays it's preferred to use endpoints (interfaces with a single method: listen) basically, rather than calling listenTCP direclty.